### PR TITLE
Add limitation to the maintenance release note

### PR DESCRIPTION
**Pull request by ZheSun88**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/platform/943)

### CODE REVIEW

File: scripts/generator/templates/template-release-notes-maintenance.md [hunk 1]
```diff
@@ -30,9 +30,20 @@ For the full list of how to get started go to the [GitHub releases](https://gith
...
+- OSGi with Bower has problems with V14.0.0 but we are working on a fix to be released in a maintenance version (V14.0.X)
+
 ## Flow
+- The Template-in-Template feature has [some limitations](https://github.com/vaadin/flow/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Atemplate-in-template+)
+- There are [some issues](https://github.com/vaadin/flow/issues/5146) in using Web Sockets as the Push channel in certain OSGi environments, but long polling works.
 
 ## Components
+- The dark theme preset does not work in IE11 when applied to the `<html>` element.
+
+## Designer
```
Comment (juhopiirainen) [inline scripts/generator/templates/template-release-notes-maintenance.md]: Does designer need to be here?

Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes-maintenance.md]: it is in the other release notes.. but honestly, i don't think it should be here. as it is not part of platform..

Comment (juhopiirainen) [reply scripts/generator/templates/template-release-notes-maintenance.md]: Yeah, it should not be there. Lets remove it.

Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes-maintenance.md]: nuked.. 💯

**Pull request merged**